testsuite/MDAnalysisTests/util.py

```diff
@@ -148,3 +149,50 @@ def in_dir(dirname):
     os.chdir(dirname)
     yield dirname
     os.chdir(old_path)
+
+
+def assert_nowarns(warning_class, *args, **kwargs):
```

Comment:
Is there a test for this function? I do not see it, but I may have missed it. Note however that @utkbansal may be working on `test_util.py` and asked for it to be frozen.